department-of-veterans-affairs/caseflow
# APPEALS 13059 - update VHA issue categories

Resolves [APPEALS-13059](https://vajira.max.gov/browse/APPEALS-13059)

### Description
Issue categories have been updated 

### Acceptance Criteria
- [x] When an intake user establishes a decision review with VHA as the Benefit Types, the following issue categories are available options
  - [x] - Beneficiary Travel
  - [x] - Clothing Allowance
  - [x] - Medical and Dental Care Reimbursement
  - [x] - Initial Eligibility and Enrollment in VHA Healthcare
  - [x] - Eligibility for Dental Treatment
  - [x] - CHAMPVA
  - [x] - Camp Lejune Family Member
  - [x] - Foreign Medical Program
  - [x] - Spina Bifida Treatment (Non-Compensation)
  - [x] - Continuing Eligibility/Income Verification Match (IVM)
  - [x] - Prosthetics | Other (not clothing allowance)

- [x] The following existing VHA issue categories should be removed as available options during intake
  - [x] - Beneficiary Travel | Common Carrier
  - [x] - Beneficiary Travel | Mileage
  - [x] - Beneficiary Travel | Special Mode
  - [x] - Clothing Allowance 3.810(b) Certification
  - [x] - Eligibility for Fee Basis Care
  - [x] - Eligibility for Treatment | Other
  - [x] - Eligibility for Treatment | Dental
  - [x] - Family Member Services | CHAMPVA Eligibility
  - [x] - Family Member Services | CHAMPVA Medical Charges
  - [x] - Family Member Services | Camp Lejune Family Member Eligibility
  - [x] - Family Member Services | Foreign Medical Program Medical Charges
  - [x] - Family Member Services | Spina Bifida Medical Charges
  - [x] - Level of Priority for Treatment
  - [x] - Prosthetics

 Services
- 

### Testing Plan
1. Go to /decision_reviews/vha and switch users to ACBAUERVVHAH
2. Copy the veteran ID or SSN from one of the rows and go to intake new form
3. Select higher level review and continue
4. Input the ID or SSN and continue
5. Fill out Benefit Type as Veterans Health Administration and put anything for the other form entries; continue
6. Click Add Issue
7. Ensure the issue categories line up with what the ticket specifies

### User Facing Changes


BEFORE
---
<img width="732" alt="image" src="https://user-images.githubusercontent.com/17840614/217074363-5a421292-58bf-4b0c-8041-6e903313cb0b.png">
<img width="718" alt="image" src="https://user-images.githubusercontent.com/17840614/217074395-dc7cfda8-82d5-4979-adfa-5e626de5ee13.png">
<img width="728" alt="image" src="https://user-images.githubusercontent.com/17840614/217074498-3e108abb-60ee-4d61-bea7-59bd35cc2f33.png">


AFTER
---
<img width="743" alt="image" src="https://user-images.githubusercontent.com/17840614/217074226-d3a8a587-a7df-4d9e-b25e-3f278833fa16.png">
<img width="740" alt="image" src="https://user-images.githubusercontent.com/17840614/217074256-399f1dbd-6f02-4b83-90e5-7021e08885b7.png">
